db/migrate/20150511140404_create_popup_notifications.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePopupNotifications < ActiveRecord::Migration
+  def change
+    create_table :popup_notifications do |t|
+      t.string :position
```

Comment:
cuz, we may have central popup or right side popup. Use position to determine which kind of popup it is.